### ISSUE TITLE
0.6.1 -> java.lang.NoSuchMethodError: com.android.ddmlib.AndroidDebugBridge.init(Lcom/android/ddmlib/AdbInitOptions;)V

### ISSUE THREAD

**Issue by Georgevik** [MarathonLabs/marathon#458]
Hi

We are using Marathon 0.6.0 in our code base. Since 0.6.1 was released, we can't update to 0.6.1 because when we run the test this error is displayed:

```
java.lang.NoSuchMethodError: com.android.ddmlib.AndroidDebugBridge.init(Lcom/android/ddmlib/AdbInitOptions;)V
```

What are we missing?

**Comment by Malinskiy** [comment]
Hey @Georgevik 

This is most likely classpath manipulation issue: we've updated ddmlib to 27.1.2 in #438 which introduced `AdbInitOptions` class in the first place.

If you're using gradle (and we've had a lot of issues with gradle's classpath before), then your only option is to force the ddmlib version in the `classpath` similar to https://github.com/Malinskiy/marathon/blob/develop/sample/android-app/build.gradle.kts#L18

This is mentioned in the [docs](https://malinskiy.github.io/marathon/doc/downloading.html#choosing-the-distribution) explicitly as a problem with gradle plugin in general. We're working on changes to architect gradle plugin in such a way that it will call to CLI which doesn't have this sort of problems.

Can you please confirm that you're using gradle plugin?

Thanks

**Comment by asos-jorgegonzalez** [comment]
Yes, we are using Android Gradle Plugin 4.0. I will force the ddmlib version using `classpath` and let you know. 

To use Gradle plugin to call CLI sounds great! Gradle sometime (most) is a headache

**Comment by Georgevik** [comment]
Yes, we are using Android Gradle Plugin 4.0. I will force the ddmlib version using classpath and let you know.

To use Gradle plugin to call CLI sounds great! Gradle sometime (most) is a headache

**Comment by Malinskiy** [comment]
@Georgevik any luck forcing the ddmlib version?

**Comment by Georgevik** [comment]
Negative. I tried to force the `ddmlib` version but I couldn't. What I did is to remove Marathon from gradle and run it using CLI. Now it is working fine

**Comment by Malinskiy** [comment]
Closed in 0.7.0 by #573
